.github/workflows/test.yml

```diff
@@ -66,27 +92,26 @@ jobs:
                      src/**/*.hs exe/*.hs
 
       - name: Build
-        shell: bash
         # Retry it three times to workaround compiler segfaults in windows
         run: cabal build || cabal build || cabal build
 
       - name: Test ghcide
-        shell: bash
+        if: ${{ !matrix.ghc-lib && matrix.test }}
```

Comment:
```suggestion
        if: ${{ matrix.test }}
```

Reply:
the plan is restore the ghc-lib build, no? do you prefer add the condition at that point?

Reply:
There is an issue for that (#784). Better to do it in a separate workflow